java/ql/lib/semmle/code/java/Expr.qll

```diff
@@ -161,18 +161,58 @@ class CompileTimeConstantExpr extends Expr {
     )
   }
 
+  /**
+   * Gets the stringified value of this expression, where possible.
+   *
+   * The stringified version of a compile-time constant expression is the equivalent to
+   * the result of calling `String.valueOf(expr)` on the expression.
+   * 
+   * Note that this does not handle the following cases:
+   *
+   * - mathematical computations of type `long`.
```

Comment:
This also doesn't work with float computations, anything else I should call out explicitly?